Pod/Classes/ResizingTextView.swift

```diff
@@ -14,6 +14,16 @@ open class ResizingTextView: UITextView {
     
     private var heightConstraint: NSLayoutConstraint!
     
+    private var maxHeight: CGFloat?
+    
+    public var resizeForNumberOfLines: Int = 0 {
```

Comment:
Couple things:
1) The name of this variable isn't very clear to me. I'd prefer `maxNumberOfLines` I think.
2) 0 isn't a great default here to disable the behavior. In Objective-C, that would be right, but in Swift, we should just make this an optional `Int?`.

Reply:
Actually, thinking about it more, if we are doing this, we should allow an explicitly set `maxHeight` as a `CGFloat` also. This should probably be an enum

````
public enum ResizableHeight {
    case height(CGFloat)
    case numberOfLines(Int)
    case infinite
}
````

Then you just have 
````
public var maxResizableHeight: ResizableHeight = .infinite
public var minResizableHeight: ResizableHeight = .numberOfLines(1)
````